packages/xchain-thorchain/__tests__/client.test.ts

```diff
@@ -1,3 +1,4 @@
+import { Coin } from '@cosmos-client/core/cjs/openapi/api'
```

Comment:
avoid  cjs and ems imports. import {proto} from core instead and use proto.cosmos.base.v1beta1.Coin